content/docs/examples/staticsite.md

```diff
@@ -0,0 +1,145 @@
+{
+    "type": "examples",
+    "layout": "type",
+    "title": "Integrate React JS",
+    "linktitle": "static sites",
+    "card_icon": "ti-infinite",
+    "card_body": "Learn how to connect React to the Static Site Resource",
+    "weight": "1"
+}
+
+# Connect React JS to the Static Site Resource
+{{<header_divider>}}
+
+Many websites are built using a front-end library or framework. `React(React.js or ReactJS)` is one of the most popular front-end libraries. `React` can easily integrate with `Cdev` to be a static site resource and complement the backend. For more information about how to use `React`, visit the **[official documentation](https:/reactjs.org/)**.
+
+{{<break 1>}}
+
+## Requirements
+To create a `React` application you will need to install `Node.js`, a JavaScript runtime, on your `Windows Subsystem for Linux`. The instructions can be found here: **[Install Node.js on Windows Subsystem for Linux(WSL)](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl)** . 
+{{<break 1>}}
+
+The initial file structure should resemble the image below. If this is not your starting point for this example, consult the `Getting Started` section to initialize a `Cdev project` before continuing.
+
+{{<tutorial_image>}}
+/images/react_example/init_img.png
+{{</tutorial_image>}}
+{{<break 2>}}
+
+## Create a React Application
+In the root of your project folder, create a `React` app with the following command:
+```bash
+npx create-react-app <your-app-name-here>
+# example
+npx create-react-app my-react-app
+```
+Once the `React` app is created, the file structure will look like this:
+{{<tutorial_image>}}
+/images/react_example/react_file_structure.png
+{{</tutorial_image>}}
+{{<break 1>}}
+
+Using the terminal, enter the folder that contains the `React` app and start the local server to confirm that the `React` app was created correctly.
+```bash
+cd <your-app-name-here>
+npm start
+
+#example 
+cd my-react-app
+npm start
+```
```

Comment:
It may be a good idea to separate code snippets that are terminal commands where each command has their own snippet so the reader can copy and paste them directly into the terminal without issue

Reply:
Changes made.